build.rs

```diff
@@ -8,12 +8,13 @@ use std::{
 };
 
 use ron::de::from_reader;
+use std::collections::HashMap;
 
 fn path(env: &str, s: &str) -> PathBuf {
     PathBuf::from(env::var(env).unwrap()).join(s)
 }
 
-fn read_template_index<P: Into<PathBuf>>(p: P) -> Vec<String> {
+fn read_template_index<P: Into<PathBuf>>(p: P) -> HashMap<String, Vec<String>> {
```

Comment:
It would be great if we could use `Project` here to make this more type-safe, but that would require splitting up the whole crate (since we can't import `Project` here).